controller/controller.js

```diff
@@ -0,0 +1,17 @@
+// c = controller, m = model, v = view, db = database
+const m = require("../model/model");
+
+// TODO: find out how error catching is meant to be, repeating .catch here seems not very DRY
+
+// /api/topics Endpoint
+exports.getApiTopics = (request, response) => {
+    m.selectTopics().then((queryResult) => {
+        const responseArray = queryResult.rows;
+        response.status(200).send(responseArray);
```

Comment:
You don't want to be sending this back as an array, it should be on a object with the key being what specifically it is